src/Microsoft.Health.Fhir.Shared.Api/Controllers/FhirController.cs

```diff
@@ -538,10 +532,8 @@ public async Task<IActionResult> ConditionalPatchFhir(string typeParameter, [Fro
             IReadOnlyList<Tuple<string, string>> conditionalParameters = GetQueriesForSearch();
             var payload = new FhirPathPatchPayload(paramsResource);
 
```

Comment:
Do we need to call SetupRequestContextWithConditionalQueryMaxParallelism(); here as well?